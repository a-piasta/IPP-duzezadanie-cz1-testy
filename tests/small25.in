getRouteDescription;693
repairRoad;zplrktragvlbfofvxg;wzkyizmfylaekqrba;5
getRouteDescription;339
repairRoad;bjrwyqwfgzxwkmskxlnwaewv;ilwjaudquiaqlciukdcq;3
repairRoad;vcqqntwebtbhfa;fcmhrasnnlp;-9
addRoad;zozrzcjmlxxxt;vcqqntwebtbhfa;22;-5
getRouteDescription;948
addRoad;vcqqntwebtbhfa;zozrzcjmlxxxt;5;-6
getRouteDescription;679
repairRoad;pzafnbrqjpua;fcmhrasnnlp;-11
removeRoad;vcqqntwebtbhfa;zozrzcjmlxxxt
addRoad;vcqqntwebtbhfa;wzkyizmfylaekqrba;8;-3
addRoad;vcqqntwebtbhfa;taypryutmbmov;13;13
repairRoad;pzafnbrqjpua;wzkyizmfylaekqrba;-2
newRoute;875;vcqqntwebtbhfa;iyowuqydhudasjitgeduyqsldqlj
newRoute;68;taypryutmbmov;zplrktragvlbfofvxg
removeRoad;zozrzcjmlxxxt;vcqqntwebtbhfa
getRouteDescription;68
extendRoute;68;taypryutmbmov
getRouteDescription;68
removeRoad;zozrzcjmlxxxt;vcqqntwebtbhfa
getRouteDescription;875
addRoad;vcqqntwebtbhfa;iyowuqydhudasjitgeduyqsldqlj;9;-11
repairRoad;ilwjaudquiaqlciukdcq;zozrzcjmlxxxt;10
repairRoad;bjrwyqwfgzxwkmskxlnwaewv;iyowuqydhudasjitgeduyqsldqlj;-12
addRoad;fcmhrasnnlp;iyowuqydhudasjitgeduyqsldqlj;13;-1
removeRoad;vcqqntwebtbhfa;zozrzcjmlxxxt
extendRoute;68;vcqqntwebtbhfa
getRouteDescription;68
removeRoad;vcqqntwebtbhfa;zozrzcjmlxxxt
removeRoad;iyowuqydhudasjitgeduyqsldqlj;vcqqntwebtbhfa
removeRoad;vcqqntwebtbhfa;iyowuqydhudasjitgeduyqsldqlj
extendRoute;875;zozrzcjmlxxxt
addRoad;taypryutmbmov;ilwjaudquiaqlciukdcq;13;1
extendRoute;68;vcqqntwebtbhfa
addRoad;vcqqntwebtbhfa;bjrwyqwfgzxwkmskxlnwaewv;9;1
removeRoad;bjrwyqwfgzxwkmskxlnwaewv;vcqqntwebtbhfa
extendRoute;68;wzkyizmfylaekqrba
addRoad;iyowuqydhudasjitgeduyqsldqlj;iyowuqydhudasjitgeduyqsldqlj;8;3
newRoute;981;zozrzcjmlxxxt;ilwjaudquiaqlciukdcq
newRoute;405;bjrwyqwfgzxwkmskxlnwaewv;pzafnbrqjpua
getRouteDescription;405
addRoad;ilwjaudquiaqlciukdcq;zozrzcjmlxxxt;7;9
repairRoad;wzkyizmfylaekqrba;vcqqntwebtbhfa;-1
repairRoad;ilwjaudquiaqlciukdcq;bjrwyqwfgzxwkmskxlnwaewv;-3
getRouteDescription;981
getRouteDescription;68
addRoad;zplrktragvlbfofvxg;fcmhrasnnlp;27;-4
getRouteDescription;875
addRoad;fcmhrasnnlp;iyowuqydhudasjitgeduyqsldqlj;11;9
repairRoad;vcqqntwebtbhfa;wzkyizmfylaekqrba;-14
extendRoute;68;wzkyizmfylaekqrba
removeRoad;vcqqntwebtbhfa;zozrzcjmlxxxt
extendRoute;50;fcmhrasnnlp
extendRoute;981;zplrktragvlbfofvxg
