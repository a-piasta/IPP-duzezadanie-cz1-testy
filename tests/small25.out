1: 
2: NIE
3: 
4: NIE
5: NIE
6: TAK
7: 
8: NIE
9: 
10: NIE
11: TAK
12: TAK
13: TAK
14: NIE
15: NIE
16: NIE
17: NIE
18: 
19: NIE
20: 
21: NIE
22: 
23: TAK
24: NIE
25: NIE
26: TAK
27: NIE
28: NIE
29: 
30: NIE
31: TAK
32: NIE
33: NIE
34: TAK
35: NIE
36: TAK
37: TAK
38: NIE
39: NIE
40: NIE
41: NIE
42: 
43: TAK
44: TAK
45: NIE
46: 
47: 
48: TAK
49: 
50: NIE
51: NIE
52: NIE
53: NIE
54: NIE
55: NIE
